iroha/src/asset.rs

```diff
@@ -212,21 +219,23 @@ pub mod query {
 
     impl Query for FindAssetById {
         #[log]
-        fn execute(&self, world_state_view: &WorldStateView) -> Result<Value, String> {
+        fn execute(&self, world_state_view: &WorldStateView) -> Result<Value> {
             Ok(world_state_view
                 .read_asset(&self.id)
                 .cloned()
-                .ok_or(format!(
-                    "Failed to get an asset with identification: {}.",
-                    &self.id
-                ))?
+                .ok_or_else(|| {
+                    Error::msg(format!(
+                        "Failed to get an asset with identification: {}.",
+                        &self.id
+                    ))
```

Comment:
Also what do you think, should we make an `error!` macro that will basically wrap arguments into `Error::msg(format!(..))`? Might be convenient.